.gitignore

```diff
@@ -2,7 +2,7 @@
 cache/
 out/
 node_modules/
-
+script/output/5/eoracle_middleware_contracts_deployment_data.json
```

Comment:
Usually we want to upload the deployment data.
If we don't want to add it in this branch let's just not add the file and remove the gitignore.

Reply:
removed